src/Checkbox/Checkbox.tsx

```diff
@@ -81,11 +81,14 @@ export default class Checkbox extends React.PureComponent<Props> {
     if (ariaCheckedState && partial) {
       ariaCheckedState = "mixed";
     }
+    const hasLabel = !!children;
 
     return (
       <button
         aria-checked={ariaCheckedState}
-        aria-labelledby={this._labelID}
+        aria-labelledby={hasLabel && this._labelID}
```

Comment:
this could be a ternary, like the aria-label below. not sure if its necessary though